### ISSUE TITLE
package.json folder incorrectly identified for each project

### ISSUE THREAD

**Issue by WHenderson** [KnodesCommunity/typedoc-plugins#21]
# Config
```js
module.exports = {
    name: '@crikey/stores-mono',
    out: 'docs',
    entryPointStrategy: 'packages',
    entryPoints: [
        'packages/*'
    ]
}
```
Approximation of file structure
```
/README.md
/package.json
/packages/a/package.json
/packages/a/README.md
/packages/b/package.json
/packages/b/README.md
```

# Result
For each project, the `README.md` from the root of the mono repo is selected instead of the `README.md` alongside the projects `package.json`.

# Cause/Solution
[plugin.ts](https://github.com/KnodesCommunity/typedoc-plugins/blob/main/packages/plugin-monorepo-readmes/src/plugin.ts#L28)
Line 28.

Change:
    `const src = u.model.sources?.[0].fileName;` (filename path is relative to the package configuration, not the project root)
to something like:
    `const src = u.model.sources?.[0].file.fullFileName;` (full absolute path)

I haven't reviewed this change in depth, so there may be a better solution, but it solves the issue for my configuration.

# Other
Also, thanks for writing these plugins. They really fill a hole in TypeDoc for me :-)

**Comment by GerkinDev** [comment]
Hi! Thank you for the report! I'll try to reproduce your error in tests suites asap, and release a fixed version. Expect it to be out on Monday or Tuesday

Have a nice weekend!

**Comment by GerkinDev** [comment]
Fixed in v0.22.3

**Comment by WHenderson** [comment]
It's working great. Thanks so much :-)
